application/src/main/java/org/togetherjava/tjbot/config/Config.java

```diff
@@ -96,6 +98,7 @@ private Config(@JsonProperty(value = "token", required = true) String token,
         this.githubApiKey = Objects.requireNonNull(githubApiKey);
         this.databasePath = Objects.requireNonNull(databasePath);
         this.projectWebsite = Objects.requireNonNull(projectWebsite);
+        this.memberCountCategoryPattern = memberCountCategoryPattern;
```

Comment:
`Objects#requireNonNull()` should also be used here if the `memberCountCategoryPattern` property becomes required.

Reply:
Ok chris 
![2x](https://github.com/Together-Java/TJ-Bot/assets/65783463/6c3d6348-706b-4b19-b015-2876f649352a)

Reply:
Done